### PR TITLE
Angular install

### PR DESCRIPTION
**Pull request by thefad3**
Added Angular CDN and also setup the main.js file and the main controller for the app's body. Also changed the formatting which Angular uses.

### CODE REVIEW

**Comment by reactivepixel** [comment]
Please move the js files out of public and into the assets main folder with appropriate sub folders.

Then setup up js uglifying through gulp

**Comment by thefad3** [comment]
Waiting for Codeship build and added changes for Gulp to serve JS files unglified

**Comment by reactivepixel** [comment]
Looks like a merge conflict, make sure to pull master into your files

**Comment by reactivepixel** [comment]
Looks like the build is still failing.

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//Global App name is ProRubric
+var appModule = angular.module('ProRubric', []);
+
+//Required to remove the conviction between Handlebars and Angular.
```
Comment (reactivepixel) [inline assets/js/main.js:4]: "Conviction?"

**Pull request merged**